docs/eta-pi-p/manual.ipynb

```diff
@@ -674,8 +888,8 @@
      "source_hidden": true
     },
     "tags": [
-     "hide-input",
-     "full-width"
+     "full-width",
+     "hide-input"
```

Comment:
Make the plot with the histograms interactive as well. Can be done in the same PR now that you know the trick ;)